src/components/PageSections/EventsHomePage/index.js

```diff
@@ -1,18 +1,10 @@
 import React from 'react';
 import { useIntl } from 'react-intl';
 import { Link } from 'react-router-dom';
-import moment from 'moment-timezone';
-
+import { filterEvents } from 'js/helpers/cleanData.js'
 import EventListEntry from 'components/Pages/EventList/EventListEntry';
 import { events as i18n } from 'js/i18n/definitions';
 
-const filterEvents = (events) => {
-  const dateNow = moment().tz('America/Chicago').format('YYYY-MM-DD')
```

Comment:
moved to cleanData.js

Reply:
Question about this - don't we still want to keep events filtering in the component itself?

Filtering at the time of clean data is good. But if we build the site one day, then visit the site 3 days later, won't filtering just at the time of creation become obsolete? Don't we need both?

Reply:
the function is being imported from cleanData, its still happening here in the component